gdb/rust-lang.c

```diff
@@ -35,7 +35,17 @@
 
 extern initialize_file_ftype _initialize_rust_language;
 
-
+/* Returns the last segment of a Rust path like foo::bar::baz.  
+   Will not handle cases where the last segment contains generics.  */
+
+static char*
+rust_last_path_segment(const char* path)
+{
+  /* This could perhaps be improved to use strtok,
+     but is good enough for our purposes.  */
```

Comment:
I think it's best to just remove this comment.  strtok can't be used here anyway.